projects/core/src/i18n/translation.service.ts

```diff
@@ -12,12 +12,12 @@ export abstract class TranslationService {
    * If key is missing, it tries to load the chunk and emits a value when chunk is loaded.
    * If key is missing after loaded chunk, a fallback value is emitted
```

Comment:
It accepts also an array of keys. It allows for providing multiple variations of a key or multiple keys that may be used interchangeably as fallback. In case the primary key is not available, the fallback keys will be attempted in the order they are provided. The first key that can be resolved will be used for the translation.

When translation value cannot be resolved, the fallback value is produced by a method `getFallbackValue`. 

```
< please add an example(s) of usage here, e.g. with fallback error keys >
```

Note: When using lazy-loading of translation chunks, the function will first load all the chunks for the corresponding keys before attempting to resolve the value. This ensures that all the necessary data is available before the translation is performed.